frontend/cypress/tests/zoomView/zoomView_hideCommitMessages.js

```diff
@@ -53,4 +53,160 @@ describe('hide all commit messages ', () => {
     cy.get('#tab-zoom .commit-message .body')
         .should('be.visible');
   });
+
+  it('check correct display of show all and hide all commit messages', () => {
```

Comment:
May I suggest that you split the hide/hide into 2 separate cases for consistency reason

Reply:
Got it, I've split them into 3 cases (only 'hide all', only 'show all' and both 'show all' & 'hide all') - I hope that's okay?